src/main/java/org/apache/maven/shared/jarsigner/JarSignerCommandLineBuilder.java

```diff
@@ -82,28 +82,28 @@ public Commandline build( JarSignerRequest request )
         }
 
         String storetype = request.getStoretype();
-        if ( !StringUtils.isEmpty( storetype ) )
+        if ( !(storetype == null || storetype.isEmpty()) )
         {
             cli.createArg().setValue( "-storetype" );
             cli.createArg().setValue( storetype );
         }
 
         String providerName = request.getProviderName();
-        if ( !StringUtils.isEmpty( providerName ) )
+        if ( !(providerName == null || providerName.isEmpty()) )
         {
             cli.createArg().setValue( "-providerName" );
             cli.createArg().setValue( providerName );
         }
 
         String providerClass = request.getProviderClass();
-        if ( !StringUtils.isEmpty( providerClass ) )
+        if ( !(providerClass == null || providerClass.isEmpty()) )
```

Comment:
No it's a bit unfortunate; would have required a separate recipe (or targeted manual change) to first replace the `!StringUtils.isEmpty` with `StringUtils.isNotEmpty`. But then since we are phasing these out we decided against it on an earlier pull request.

In this specific case we can fix these manually here if you'd like, as I imagine checkstyle will have it's complaints as well.